test/acceptance/framework/consul/consul_cluster.go

```diff
@@ -149,7 +149,7 @@ func (h *HelmCluster) Destroy(t *testing.T) {
 	}
 
 	// Delete any secrets that have h.releaseName in their name.
-	secrets, err := h.kubernetesClient.CoreV1().Secrets(h.helmOptions.KubectlOptions.Namespace).List(context.Background(), metav1.ListOptions{LabelSelector: "release=" + h.releaseName})
+	secrets, err := h.kubernetesClient.CoreV1().Secrets(h.helmOptions.KubectlOptions.Namespace).List(context.Background(), metav1.ListOptions{})
```

Comment:
This is a bug that leaves secrets around after test runs. We don't add release labels to the secrets we create in the helm chart jobs.